### PR TITLE
feature: Remove mobx dependency

### PR DESCRIPTION
**Pull request by otothea**

### CODE REVIEW

File: src/keyboard-shortcuts.tsx [hunk 1]
```diff
@@ -11,20 +9,21 @@ interface IProps {
   shortcuts: { [chord: string]: () => void } | Shortcut[];
 }
 
-@observer
 export class KeyboardShortcuts extends Component<IProps> {
-  shortcuts: {};
-  @observable layer: ShortcutLayer;
+  shortcuts: Shortcut;
```
Comment (tarwich) [inline src/keyboard-shortcuts.tsx]: I'm not sure, but it seems like the type here would be:

```ts
shortcuts: Record<string, Shortcut> | Shortcut[];
```

That being said... Is this ever read, or only written?

Comment (otothea) [reply src/keyboard-shortcuts.tsx]: actually, it is neither read to or written to. Removed it

**Review by tarwich — COMMENTED** [review]
I'm blown away that this actually works. Splendid!

File: src/shortcut-manager.tsx [hunk 2]
```diff
@@ -192,6 +194,8 @@ export class ShortcutManager {
         this.store.modifiers[modifier] = false;
       }
     );
+
+    installDebug && installDebug.forceUpdate();
```
Comment (tarwich) [inline src/shortcut-manager.tsx:203]: image

The symbols at the top right should highlight on keydown.

File: package.json [hunk 3]
```diff
@@ -3,13 +3,14 @@
...
+  "typescript": "index.ts",
   "dependencies": {},
   "devDependencies": {
+    "@types/lodash": "4.14.136",
     "@types/react-dom": "16.0.11",
     "eslint": "^5.10.0",
     "eslint-config-tarwich": "github:tarwich/eslint-config-tarwich",
-    "mobx": "5.6.0",
-    "mobx-react": "^5.4.3",
+    "lodash": "4.17.14",
```
Comment (tarwich) [inline package.json:13]: Have you done an `npm link` and made sure this runs in our project?

Comment (otothea) [reply package.json:13]: yes i am testing it and running it in our project

File: src/shortcut-manager.tsx (hunk 2 above)
Comment (otothea) [reply src/shortcut-manager.tsx:203]: Fixed

**Review by tarwich — APPROVED** [review]

**Pull request merged**